src/Serializer/AbstractItemNormalizer.php

```diff
@@ -331,7 +331,16 @@ protected function getAttributeValue($object, $attribute, $format = null, array
     {
         $propertyMetadata = $this->propertyMetadataFactory->create($context['resource_class'], $attribute, $this->getFactoryOptions($context));
 
-        $attributeValue = $this->propertyAccessor->getValue($object, $attribute);
+        try {
+            $attributeValue = $this->propertyAccessor->getValue($object, $attribute);
+        } catch (NoSuchPropertyException $e) {
+            if (null === $propertyMetadata->isChildInherited()) {
+                throw $e;
```

Comment:
Take 2 entities:

- AbstractParentEntity
- ChildOne having `fooProp` extending AbstractParentEntity

On `AbstractParentEntity`, no getters/setters are available for the `fooProp` and `propertyAccessor` will throw if the wanted property is not in the `$object` (here `fooProp`). 
This only happens when we try to fetch `fooProp` and it is not available directly.